Examples/CaseStudies/SwiftUICaseStudies/02-Effects-SystemEnvironment.swift

```diff
@@ -203,23 +208,23 @@ struct SystemEnvironment<Environment> {
 }
 
 #if DEBUG
-  import XCTestDynamicOverlay
```

Comment:
Sorry! Nothing changed here, I'm using different rules for `#if Debug` indentation in my `swift-format` Xcode extension.